kazuki-toolkit/src/main/kotlin/com/anaplan/engineering/kazuki/toolkit/ISO8601/Constants.kt

```diff
@@ -0,0 +1,59 @@
+package com.anaplan.engineering.kazuki.toolkit.ISO8601
+
+import com.anaplan.engineering.kazuki.core.*
+import com.anaplan.engineering.kazuki.toolkit.ISO8601.Date_Module.mk_Date
+import com.anaplan.engineering.kazuki.toolkit.ISO8601.Dtg_Module.mk_Dtg
+import com.anaplan.engineering.kazuki.toolkit.ISO8601.Time_Module.mk_Time
+
+enum class PlusOrMinus {
+    Plus,
+    Minus,
+    None
+}
+
+typealias OffsetDirection = PlusOrMinus
+
+
+enum class DayOfWeek { // Date.functions.toDayOfWeek depends on this ordering - don't change it!
```

Comment:
:bulb: It shouldn't be necessary to have a comment like this, the behaviour should be protected by a test/AC